<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>progresswindow</class>
 <widget class="QDialog" name="progresswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parsing Information</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>366</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>406</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>View Folder</string>
   </property>
  </widget>
  <widget class="QLabel" name="articles_parsed_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Articles Parsed</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_articles_parsed">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="done_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>343</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="good_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Good Articles</string>
   </property>
  </widget>
  <widget class="QLabel" name="good_title_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bad Articles</string>
   </property>
  </widget>
  <widget class="QLabel" name="good_title_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Regular Articles</string>
   </property>
  </widget>
  <widget class="QLabel" name="good_title_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Redirect Articles</string>
   </property>
  </widget>
  <widget class="QLabel" name="good_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bad_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="reg_label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="redir_label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>25</y>
     <width>221</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="articles_parsed_label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>198</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Bytes Processed</string>
   </property>
  </widget>
  <widget class="QLabel" name="bytes_processed">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>198</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
